packages/components/helptext/README.mdx

```diff
@@ -1,28 +1,31 @@
+# @contentful/f36-helptext
+
+This package is part of [Forma 36](https://github.com/contentful/forma-36).
+
```

Comment:
I don't think we can have this before the YAML frontmatter, it's probably not valid markdown. The website doesn't build with it: https://deploy-preview-953--forma-36.netlify.app/components/help-text/

Reply:
✅  removed